### PR TITLE
Documentation: Added kubeadm documentation #618

### PR DESCRIPTION
**Pull request by loganasherjones**
This getting started guide covers a simple deployment of kube-prometheus
and assumes you have deployed using kubeadm.

Hopefully this helps clears up #618 

I went ahead and took a stab at creating some initial documentation for how to get `kube-prometheus` up and running after installing a cluster with `kubeadm`. Any feedback is welcome. 

I wasn't exactly sure where I should put the documentation, so if there is a better place to put this, just let me know and I can move it around.

### CODE REVIEW

**Comment by brancz** [comment]
We currently separate the kube-prometheus docs in https://github.com/coreos/prometheus-operator/blob/master/contrib/kube-prometheus/docs/ . I think this would be a good case for this as well. Otherwise this looks great!

**Comment by loganasherjones** [comment]
Ok, I moved the files over to `contrib/kube-prometheus/docs` and updated the links. Let me know if you see any other problems!

File: contrib/kube-prometheus/docs/kube-prometheus-on-kubeadm.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+```
+find manifests/prometheus -type f ! -name prometheus-k8s-roles.yaml ! -name prometheus-k8s-role-bindings.yaml -exec kubectl --namespace "$NAMESPACE" apply -f {} \;
+kubectl apply -f manifests/prometheus/prometheus-k8s-roles.yaml
+kubectl apply -f manifests/prometheus/prometheus-k8s-role-bindings.yaml
+```
+
+Finally, install the [AlertManager](../../../Documentation/user-guides/alerting.md)
+
+```
+kubectl --namespace="$NAMESPACE" apply -f manifests/AlertManager
```
Comment (brancz) [inline contrib/kube-prometheus/docs/kube-prometheus-on-kubeadm.md]: `manifests/AlertManager` -> `manifests/alertmanager`

File: contrib/kube-prometheus/docs/kube-prometheus-on-kubeadm.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+Next up is the `Prometheus` object itself. We will deploy the application, and then the roles/role-bindings.
+
+```
+find manifests/prometheus -type f ! -name prometheus-k8s-roles.yaml ! -name prometheus-k8s-role-bindings.yaml -exec kubectl --namespace "$NAMESPACE" apply -f {} \;
+kubectl apply -f manifests/prometheus/prometheus-k8s-roles.yaml
+kubectl apply -f manifests/prometheus/prometheus-k8s-role-bindings.yaml
+```
+
+Finally, install the [AlertManager](../../../Documentation/user-guides/alerting.md)
```
Comment (brancz) [inline contrib/kube-prometheus/docs/kube-prometheus-on-kubeadm.md]: For the better or worse the project is referred to as "Alertmanager"

File: contrib/kube-prometheus/docs/kube-prometheus-on-kubeadm.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+```
+
+Next, we will install the node exporter and then kube-state-metrics:
+
+```
+kubectl --namespace="$NAMESPACE" apply -f manifests/node-exporter
+kubectl --namespace="$NAMESPACE" apply -f manifests/kube-state-metrics
+```
+
+Then, we can deploy the grafana credentials. By default, the username/password will be `admin/admin`, you can feel free to change these for your production clusters.
```
Comment (brancz) [inline contrib/kube-prometheus/docs/kube-prometheus-on-kubeadm.md]: This is highly encouraged :slightly_smiling_face:

**Comment by loganasherjones** [comment]
Great, I can fix these, then do you want me to squash the commits into a single commit?

**Comment by brancz** [comment]
This lgtm, thanks a lot!

**Pull request merged**